rel-eng/build-packages-for-obs.sh

```diff
@@ -85,6 +85,34 @@ FAILED_PKG=
 VERBOSE=$VERBOSE
 while read PKG_NAME PKG_VER PKG_DIR; do
  for tries in 1 2 3; do
+
+  if [[ $PKG_DIR == *"containers"* ]]; then
+    CONTAINER_NAME=$(basename "$PKG_DIR")
+    echo "=== Building container image [${CONTAINER_NAME}]"
+    if [ -d "$PKG_DIR" ]; then
+      cp -r "$PKG_DIR" "$SRPM_DIR/"
+      if [ -f "${PKG_DIR}/Chart.yaml" ]; then
+        pushd "${SRPM_DIR}/${CONTAINER_NAME}"
+        CHART_FILES="values.yaml values.schema.json charts crds templates LICENSE README.md"
+        TO_INCLUDE=""
+        for F in ${CHART_FILES}; do
+          if [ -e "${F}" ]; then
+              TO_INCLUDE="${TO_INCLUDE} ${F}"
+          fi
+        done
+        tar cf "${SRPM_DIR}/${CONTAINER_NAME}/${CONTAINER_NAME}.tar" "${TO_INCLUDE}"
+        rm -r "${TO_INCLUDE}"
```

Comment:
```suggestion
        rm -r ${TO_INCLUDE}
```
See https://github.com/SUSE/spacewalk/issues/20345#issuecomment-1501619758